distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -74,13 +78,23 @@ protected void printAdditionalHelp(Terminal terminal) {
 
     @Override
     protected void execute(Terminal terminal, OptionSet options, Environment env) throws Exception {
+        final Path pluginsDescriptor = env.configFile().resolve("elasticsearch-plugins.yml");
+        if (Files.exists(pluginsDescriptor)) {
+            throw new UserException(
+                ExitCodes.USAGE,
+                "Plugins descriptor [" + pluginsDescriptor + "] exists, please use [elasticsearch-plugin sync] instead"
```

Comment:
Should we use the term "descriptor" or "config" here? I'm leaning toward the latter as the term "descriptor" is already a bit overloaded.